frontend/pages/createTournament.njk

```diff
@@ -5,46 +5,18 @@
 <hr style="width:80%; margin: auto;">
 <p style="text-align: center; margin-top: 2vh;">Create a Tournament and share the results online!</p>
 <div class="divForHome2" style="height: 80vh">
-    <p style="display: inline-block"> Tournament Name:</p>
-    <input type="name" class="form-control" id="tournamentName" placeholder="Name">
-    <p>Format: </p>
-    <div id="bla"> 
-        <div class="form-check">
-            <input class="form-check-input" type="radio" name="Format" id="formatCheckBox1">
-            <label class="form-check-label" for="formatCheckBox1">
-                Draw
-            </label>
+    <form action="http://localhost:8000/tournaments" method="post">
```

Comment:
Next step: refactor the form submission into a JavaScript fetch call so that you can dynamically navigate to the created tournament route in your frontend once you got the successful response from the API.